webapp/blog/views.py

```diff
@@ -234,28 +41,35 @@ def snap_posts(snap):
                     }
                 )
 
-    return flask.jsonify(articles)
-
-
-@blog.route("/api/series/<series>")
-def snap_series(series):
-    blog_articles = None
-    articles = []
+        return flask.jsonify(articles)
 
-    try:
-        blog_articles, total_pages = api.get_articles(series)
-    except ApiError:
-        blog_articles = []
+    @app.route(url_prefix + "/api/series/<series>")
+    def snap_series(series):
+        blog_articles = None
+        articles = []
 
-    for article in blog_articles:
-        transformed_article = logic.transform_article(
-            article, featured_image=None, author=None
-        )
-        articles.append(
-            {
-                "slug": transformed_article["slug"],
-                "title": transformed_article["title"]["rendered"],
-            }
+        try:
+            blog_articles, total_pages = wordpress_api.get_articles(series)
+        except ApiError:
```

Comment:
I think `ApiError` won't work here, since `wordpress_api` does not raise this exception.